src/card.css

```diff
@@ -0,0 +1,92 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  text-decoration: none;
```

Comment:
don't reset `margins` and `text-decoration` for all (`*`) elements
also move these styles to `reset.css / normalize.css`